client/app/components/queries/ScheduleDialog.jsx

```diff
@@ -139,6 +139,29 @@ class ScheduleDialog extends React.Component {
     this.setScheduleUntil(null, date);
   };
 
+  getTimeSelector = () => {
+    const { hour, minute, newSchedule: { time: utc } } = this.state;
```

Comment:
What this `newSchedule: { time: utc }` is about?

Reply:
`newSchedule` is the actual data saved in DB, in which `time` is kept in utc.
https://github.com/getredash/redash/blob/ade3cc72a70d2c65b6f8e393304f4a394f61e926/client/app/components/queries/ScheduleDialog.jsx#L74-L80

Reply:
👍 I got confused by the object deconstruction here.